### PR TITLE
Add external collision protection enabled srv

### PR DESCRIPTION
**Pull request by kochigami**
These two srv files are used to execute set / getExternalCollisionProtectionEnabled by using service.
1. GetExternalCollisionProtectionEnabled.srv :
   We set 
   `std_msgs/String name` : `“All”, “Move”, “Arms”, “LArm” or “RArm”`
2. SetExternalCollisionProtectionEnabled.srv:
   We set
   `std_msgs/String name` : `“All”, “Move”, “Arms”, “LArm” or “RArm”`
   `bool status` : `True or False` (which means enabled or disabled)                     

Links of two functions are here:
[getExternalCollisionPretectionEnabled](http://doc.aldebaran.com/2-1/naoqi/motion/reflexes-external-collision-api.html?highlight=getexternalcollision#ALMotionProxy::getExternalCollisionProtectionEnabled__ssCR)
[setExternalCollisionProtectionEnabled](http://doc.aldebaran.com/2-1/naoqi/motion/reflexes-external-collision-api.html?highlight=setexternalcollision#ALMotionProxy::setExternalCollisionProtectionEnabled__ssCR.bCR)

If there is any problem, please let me know.

### CODE REVIEW

**Comment by vrabaud** [comment]
If the set is limited, shouldn't you use enums ? (maybe you want to keep flexibility, I don't know)

**Comment by kochigami** [comment]
Thank you very much for your advice and I'm sorry for my late reply.

I added `ExternalCollisionProtectionNameList.msg`, which has a set of All, Move, Arms, LArm, RArm as numbers.
Then, I modified `Set/ GetExternalCollisionProtectionEnabled.srv` to include this msg.

I might misunderstand what you said. If so, I am very sorry, but I'd like to try it one more time.

File: msg/ExternalCollisionProtectionNameList.msg [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# A message for setExternalCollisionProtectionEnabled in External-Collision avoidance API
+
+uint8 number
```
Comment (suryaambrose) [inline msg/ExternalCollisionProtectionNameList.msg]: I would prefer either the usual "data" used in ROS, or a name related to the information you transmit (for instance "protected_parts"). But "data" would really be the best here I think. 
I find "number" misleading (one might think that it is the number of protected parts or whatever).

Comment (suryaambrose) [inline msg/ExternalCollisionProtectionNameList.msg]: Besides I am confused with the message name (NameList, but the message is not a List and does not contain any name)

**Comment by suryaambrose** [comment]
It could be more interesting to have a super enum with all body parts (BodyParts.msg ?) on which you could rely (your service would ask for a BodyPart, enable or disable the external collision protection if possible, return an error if not possible). 
@vrabaud what do you think ?

File: srv/GetExternalCollisionProtectionEnabled.srv [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Get information about External Collision Protection Enabled
+naoqi_bridge_msgs/ExternalCollisionProtectionNameList name
```
Comment (vrabaud) [inline srv/GetExternalCollisionProtectionEnabled.srv]: you need [] if you want a list. But replace NameList by Names for simplicity I guess.

**Comment by kochigami** [comment]
Thank you very much for your advice.

I modified from `msg/ExternalCollisionProtectionNameList.msg` to `msg/ExternalCollisionProtectionNames.msg`, and
 from `uint8 number` to `uint8 data` in it.
In addtion, I added `bool status` output (which shows whether `setExternalCollisionProtectionEnabled` succeeds or not) to `srv/SetExternalCollisionProtectionEnabled.srv`.
This is examples of working of `rosservice call /set_external_collision_protection_status`. 

```
rosservice call /set_external_collision_protection_status "name:
  data: 0
status: false"        
status: False (It failed because we need the deactivation of Pepper's safety reflex.)

 rosservice call /set_external_collision_protection_status "name:
  data: 3
status: True" 
status: True
```

If there is any problem, please let me now.

**Pull request closed without merging**